policyengine_us/variables/gov/local/ny/nyc/tax/income/credits/household/nyc_household_credit.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import *
+
+
+class nyc_household_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NYC Household Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "in_nyc"
+
+    def formula(tax_unit, period, parameters):
+        # First get their federal AGI.
+        # Technically based on recomputed AGI, which deviates slightly from federal AGI.
+        federal_agi = tax_unit("adjusted_gross_income", period)
+
+        # Then get their filing status.
+        filing_status = tax_unit("filing_status", period)
+
+        # Then get the household credit part of the parameter tree.
+        p = parameters(period).gov.local.ny.nyc.tax.income.credits.household
+
+        # Then get their number of dependents.
+        tax_unit_size = tax_unit("tax_unit_size", period)
+
+        filing_statuses = filing_status.possible_values
+
+        return select(
+            [
+                filing_status == filing_statuses.SINGLE,
+                filing_status == filing_statuses.SEPARATE,
+            ],
+            [
+                # Single filers get a flat amount.
+                p.flat_amount.calc(federal_agi),
+                # Separate filers get an amount for each person in the tax
+                # unit, varying with AGI.
+                p.separate_per_dependent.calc(federal_agi) * tax_unit_size,
```

Comment:
And below, after adjusting the parameters
```suggestion
                p.separate_per_dependent.calc(federal_agi, right=True) * tax_unit_size,
```